.github/workflows/test.yml

```diff
@@ -26,10 +26,20 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Install deps
-        run: |
-          pip install 'cylc-flow[empy] @ git+https://github.com/cylc/cylc-flow@master'
-          pip install 'metomi-rose @ git+https://github.com/metomi/rose@master'
+      # - name: Get Graphviz
+      #   run: |
+      #     conda install graphviz
+      #     conda install pygraphviz
+
+      - name: install libs
+        uses: cylc/release-actions/install-cylc-components@v1
+        with:
+          cylc_flow: true
+          cylc_flow_tag: ${{ inputs.cylc-flow-tag }}
```

Comment:
This workflow doesn't have an input called `cylc-flow-tag` so this is doing nothing.

```suggestion
```